tests/test_transforms.py

```diff
@@ -85,3 +86,42 @@ def test_transform_batch(self):
             self.assertIn('name', item)
             self.assertIn('new_field', item)
             self.assertEqual(item['new_field'], item['country_code'] + '-' + item['name'])
+
+
+class FlatsonTransformTest(unittest.TestCase):
+    def setUp(self):
+        self.options = {
+            'exporter_options': {
+                'log_level': 'DEBUG',
+                'logger_name': 'export-pipeline'
+            },
+        }
+
+        self.batch = [
+            BaseRecord({'name': 'item1', 'age': '10'}),
+            BaseRecord({'name': 'item2', 'age': '22'}),
+            BaseRecord({'name': 'item3', 'age': '49'})
```

Comment:
As it is, these objects aren't going to be changed by the transform because they're already flat.
This batch should have some nested data to be actually useful -- and the assertions should look at the contents, not just sizes.